src/Symfony/Bridge/Monolog/Handler/ElasticsearchLogstashHandler.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Symfony\Bridge\Monolog\Handler;
+
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Formatter\LogstashFormatter;
+use Monolog\Handler\AbstractProcessingHandler;
+use Monolog\Logger;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * Push logs directly to Elasticsearch and format them according to Logstash specification
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class ElasticsearchLogstashHandler extends AbstractProcessingHandler
+{
+    private $endpoint;
+    private $index;
+    private $client;
+    private $responses;
+
+    public function __construct($endpoint = 'http://elasticsearch:9200', $index = 'monolog', HttpClientInterface $client = null, $level = Logger::DEBUG, $bubble = true)
```

Comment:
missing `string` types I suppose?

Reply:
Yes, indeed. I removed them for monolog 1.x